pkg/util/util.go

```diff
@@ -83,6 +88,31 @@ func IsResourceIncluded(resource string, backup *velerov1.Backup) bool {
 	return false
 }
 
+func IsResourceExcluded(resource string, backup *velerov1.Backup) bool {
+	if len(backup.Spec.ExcludedResources) == 0 {
```

Comment:
backup can be null here, and in other functions in this util

Reply:
I think Velero guaranties that backup (and restore) objects are not nil, but it won't hurt to add a check for that at the plugin entry points (`Execute` functions).